app/src/main/java/com/zionhuang/music/ui/component/BottomSheet.kt

```diff
@@ -79,10 +81,7 @@ fun BottomSheet(
         if (!state.isCollapsed) {
             BoxWithConstraints(
                 modifier = Modifier
-                    .fillMaxSize()
-                    .graphicsLayer {
```

Comment:
I removed that because I thought the overlapping colored box fading out is enough for the expanded content to fade in. But so it's more crossfading, so maybe those lines should be added again for the fade in/out to be one after another?